src/App.jsx

```diff
@@ -1,6 +1,9 @@
 import React, { useState } from "react";
-import LandingPage from "./components/landingPage";
+// import LandingPage from "./components/landingPage";
 import Navbar from "./components/navbar";
+import Dashboard from "./components/Dashboard";
+import { Switch, Route, BrowserRouter } from "react-router-dom";
+
 // import Requests from "./components/Requests";
 
```

Comment:
Please remove comments of imports if they are not being used

Reply:
Unused comments removed!